functests/1_performance/cpu_management.go

```diff
@@ -282,32 +309,36 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", func() {
 			output, err := nodes.ExecCommandOnNode(cmd, workerRTNode)
 			Expect(err).ToNot(HaveOccurred())
 
-			cpu, err := strconv.Atoi(output)
+			cpus, err := cpuset.Parse(output)
 			Expect(err).ToNot(HaveOccurred())
 
 			By("Getting the CPU scheduling flags")
-			flags, err := getCPUSchedulingDomainFlags(cpu)
+			flags, err := getCPUsSchedulingDomainFlags()
 			Expect(err).ToNot(HaveOccurred())
 
 			By("Verifying that the CPU load balancing was disabled")
-			Expect(len(flags)).To(Equal(len(defaultFlags)))
-			// the CPU flags should be almost the same except the LSB that should be disabled
-			for i := range flags {
-				Expect(flags[i]).To(Equal(defaultFlags[i] - 1))
+			for _, cpu := range cpus.ToSlice() {
+				Expect(len(flags[cpu])).To(Equal(len(defaultFlags[cpu])))
+				// the CPU flags should be almost the same except the LSB that should be disabled
```

Comment:
Can you explain why the LSB should be disabled? (in comment)

Reply:
I will add the link to the kernel code

Reply:
done